spring-batch-docs/asciidoc/common-patterns.adoc

```diff
@@ -445,10 +445,11 @@ public MultiLineTradeItemReader itemReader() {
 public FlatFileItemReader flatFileItemReader() {
 	FlatFileItemReader<Trade> reader = new FlatFileItemReaderBuilder<Trade>()
 			.name("flatFileItemReader")
-			.resource(new ClasspathResource("data/iosample/input/multiLine.txt"))
+			.resource(new ClassPathResource("data/iosample/input/multiLine.txt"))
 			.lineTokenizer(orderFileTokenizer())
-			.fieldSetMapper(new PassThroughFieldSetMapper())
```

Comment:
This snippet does not compile. Since it is an example, I used a method call like the "orderFileTokenizer()"

Reply:
Indeed, the `PassThroughFieldSetMapper` exists and can be created with a default constructor. The issue is that the method `fieldSetMapper` of the builder in this case is expecting a field set mapper that maps a `FieldSet` to a `Trade` whereas the `PassThroughFieldSetMapper` returns a `FieldSet`.